common/djangoapps/student/views.py

```diff
@@ -1236,9 +1236,27 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
                 AUDIT_LOG.warning(u"Login failed - password for user.id: {0} is invalid".format(loggable_id))
             else:
                 AUDIT_LOG.warning(u"Login failed - password for {0} is invalid".format(email))
+
+        # Labster changes
+        # If there was login error, suggest student to login to region server
+        regions = configuration_helpers.get_value('REGIONS', settings.REGIONS)
+        current_region = request.session.get('country_code')
```

Comment:
@lukyfebrianto please leave a comment with a link to a list of Country Codes https://dev.maxmind.com/geoip/legacy/codes/iso3166/

Reply:
added